app/javascript/packs/application-common.js

```diff
@@ -31,3 +33,9 @@ ManageIQ.component = {
 ManageIQ.angular.rxSubject = rxSubject;
 window.sendDataWithRx = sendDataWithRx;
 window.listenToRx = listenToRx;
+
+$(() => {
+  newRegistry.define('SnapshotForm', reactBlueprint(props => (
```

Comment:
why do you want to mount this on every page? imho the rails view should be trigging mounting the component instead?